src/evict/evict_lru.c

```diff
@@ -411,46 +411,54 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
 		__wt_readunlock(session, &conn->dhandle_lock);
 		WT_RET(ret);
 
-		cache->pages_evicted = 0;
```

Comment:
I'm concerned we lose something dropping this part of the change. I *believe* that this call to reset pages_evicted meant that we tracked if progress was being made between calls to the `__evict_server` function. That is valuable because the eviction server does this "gather candidates" pass, and then often pauses for a while or does some eviction between passes - which gives other threads a chance to do eviction.

Reply:
Yes, I think you're correct -- I missed that.

My change would mean we only decide we "did work" if pages were pushed out (by any thread), during the time the thread executing `__evict_server()` started, and called `__evict_pass()`.